tests/test-dirs/motion/jump_match.t

```diff
@@ -0,0 +1,132 @@
+Test if location of next case is given
+  $ $MERLIN single jump -target match-next-case -position 3:3 -filename main.ml << EOF
+  > let find_vowel x = 
+  > match x with 
+  > | 'A' -> 
+  >   true
+  > | 'E' -> 
+  >   true
+  > | 'I' -> 
+  >   true
+  > | 'O' -> 
+  >   true 
+  > | 'U' -> 
+  >   true 
+  > | _ -> 
+  >   false
+  > EOF
+  {
+    "class": "return",
+    "value": {
+      "pos": {
+        "line": 5,
+        "col": 2
+      }
+    },
+    "notifications": []
+  }
+
+Test if location of prev case is given
+  $ $MERLIN single jump -target match-prev-case -position 5:2 -filename test.ml << EOF
+  > let find_vowel x = 
+  > match x with 
+  > | 'A' -> 
+  >   true
+  > | 'E' -> 
+  >   true
+  > | 'I' -> 
+  >   true
+  > | 'O' -> 
+  >   true 
+  > | 'U' -> 
+  >   true 
+  > | _ -> 
+  >   false
+  > EOF
+  {
+    "class": "return",
+    "value": {
+      "pos": {
+        "line": 3,
+        "col": 2
+      }
+    },
+    "notifications": []
+  }
+
+Test when cursor is not in a match statement
+  $ $MERLIN single jump -target match-prev-case -position 1:2 -filename test.ml << EOF
+  > let find_vowel x = 
+  > match x with 
+  > | 'A' -> 
+  >   true
+  > | 'E' -> 
+  >   true
+  > | 'I' -> 
+  >   true
+  > | 'O' -> 
+  >   true 
+  > | 'U' -> 
+  >   true 
+  > | _ -> 
+  >   false
+  > EOF
```

Comment:
Since the program is always the same, there is no reason to repeat it for each test, you can write it to a file once and for all at the beginning:
```
  $ cat >test.ml << EOF
  > let find_vowel x = 
  > match x with 
  > | 'A' -> 
  >   true
  > | 'E' -> 
  >   true
  > | 'I' -> 
  >   true
  > | 'O' -> 
  >   true 
  > | 'U' -> 
  >   true 
  > | _ -> 
  >   false
  > EOF
  
  $MERLIN single jump -target match-prev-case -position 1:2 -filename test.ml <test.ml
  ...
```